kitsune/community/templates/community/email/first_answer.html

```diff
@@ -23,7 +23,7 @@
   <p><strong>{{ _('Tell us about yourself!') }}</strong></p>
 
   <p>
-    {% trans intro_url=host + '/en-US/forums/contributors', irc_url='https://client01.chat.mibbit.com/?server=irc.mozilla.org&channel=#sumo' %}
+    {% trans intro_url='https://' + host + '/forums/contributors', irc_url='https://client01.chat.mibbit.com/?server=irc.mozilla.org&channel=#sumo' %}
```

Comment:
Yeah you had a 'relative' URL due to the missing `https://` so it tacked on the host again. This is fine or you could ditch the host altogether.